docs/toolbox/cluster.rst

```diff
@@ -26,3 +26,17 @@ Cluster Scale
     # even when there are some machinesets that might need to be downscaled
     # to 0 to achive that.
     ./run_toolbox.py cluster set_scale g4dn.xlarge 5 --force
+
+ .. code-block:: shell
+
+    # list the machinesets of the cluster
+    $ oc get machinesets -n openshift-machine-api
+
+    NAME                                      DESIRED   CURRENT   READY   AVAILABLE   AGE
+    <cluster>-8p9vm-worker-eu-central-1a      1         1         1       1           57m
```

Comment:
Since this is already an example, might as well put the actual cluster name in there, I think it makes things less abstract / confusing

Reply:
changed `<cluster>` to `playground`